bip-0388.mediawiki

```diff
@@ -151,7 +151,7 @@ See [[bip-0379.md|BIP-379]] for a precise specification of all the valid miniscr
 
 
 <tt>KEY</tt> expressions consist of
-* a <tt>KP</tt> expressions
+* a <tt>KP</tt> expression
```

Comment:
nit, would be clearer to make this change in the first commit a3928806b93bdf83

Reply:
Done